### PR TITLE
feat(queue): More logging and with objects 

### PR DESCRIPTION
**Pull request by AndesKrrrrrrrrrrr**
Improve logging for user-notifications

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/services/user-notification/src/app/modules/notifications/notifications.controller.ts [hunk 1]
```diff
@@ -91,6 +96,9 @@ export class NotificationsController {
   async getNotificationTemplates(
     @Query('locale') locale: string,
   ): Promise<HnippTemplate[]> {
+    this.logger.info(
+      `Fetching templates from Contentful GQL for locale: ${locale}`,
+    )
```
Comment (rafnarnason) [inline apps/services/user-notification/src/app/modules/notifications/notifications.controller.ts:101]: fyi - this is already logged verbatim like this in notifications.service.ts, consider another message if put in the controller for other purposes

File: apps/services/user-profile/src/app/user-profile/userToken.controller.ts [hunk 2]
```diff
@@ -52,9 +58,11 @@ export class UserTokenController {
       nationalId,
     )
     if (!userProfile) {
-      throw new NotFoundException(
+      this.logger.info(
```
Comment (rafnarnason) [inline apps/services/user-profile/src/app/user-profile/userToken.controller.ts:61]: probably missing this here
import { LOGGER_PROVIDER, type Logger } from '@island.is/logging'

**Review by rafnarnason — COMMENTED** [review]
minor comments, minor fix proposal

File: apps/services/user-notification/src/app/modules/notifications/notifications.controller.ts (hunk 1 above)
Comment (AndesKrrrrrrrrrrr) [inline apps/services/user-notification/src/app/modules/notifications/notifications.controller.ts:101]:
```suggestion
    this.logger.info(
      `Fetching hnipp template for locale: ${locale}`,
    )
```

Comment (AndesKrrrrrrrrrrr) [reply apps/services/user-notification/src/app/modules/notifications/notifications.controller.ts:101]: Updated. I'm considering whether this log is even useful :thinking:

File: apps/services/user-profile/src/app/user-profile/userToken.controller.ts (hunk 2 above)
Comment (AndesKrrrrrrrrrrr) [reply apps/services/user-profile/src/app/user-profile/userToken.controller.ts:54]: Fixed :heart:

**Comment by andes-it** [comment]
This PR's had a 15-day siesta. Lost its spark, did it? It's making an exit in 5 days unless the stale label disappears.

**Comment by andes-it** [comment]
This PR's had a 15-day siesta. Lost its spark, did it? It's making an exit in 5 days unless the stale label disappears.

File: apps/services/user-profile/src/app/user-profile/pipes/userProfileByNationalId.pipe.ts [hunk 3]
```diff
@@ -1,21 +1,33 @@
...
+    private readonly userProfileService: UserProfileService,
+  ) {}
 
   async transform(nationalId: string): Promise<UserProfile> {
     const userProfile = await this.userProfileService.findByNationalId(
       nationalId,
     )
     if (!userProfile) {
-      throw new NotFoundException(
+      this.userProfileService.logger.info(
```
Comment (saevarma) [inline apps/services/user-profile/src/app/user-profile/pipes/userProfileByNationalId.pipe.ts:26]:
```suggestion
      this.logger.info(
```

**Review by saevarma — COMMENTED** [review]
@AndesKrrrrrrrrrrr @rafnarnason  do we want to finish this one?

**Review by rafnarnason — APPROVED** [review]
lgtm

**Pull request merged**